app/tests/serving_tests/test_models.py

```diff
@@ -75,64 +76,83 @@ def test_get_component_interface_values_for_user():
 
     ci1, ci2 = ComponentInterfaceFactory.create_batch(2)
 
-    civ1, civ2, civ3, civ4, civ5, civ6, civ7, civ8 = (
-        ComponentInterfaceValueFactory.create_batch(8, interface=ci1)
+    civa1, civa2, civa3, civa4, civa5, civa6, civa7, civa8, civa9, civa10 = (
+        ComponentInterfaceValueFactory.create_batch(10, interface=ci1)
     )
-    civ9, civ10, civ11, civ12, civ13, civ14, civ15, civ16 = (
-        ComponentInterfaceValueFactory.create_batch(8, interface=ci2)
+    civb1, civb2, civb3, civb4, civb5, civb6, civb7, civb8, civb9, civb10 = (
+        ComponentInterfaceValueFactory.create_batch(10, interface=ci2)
     )
 
     job_with_perm = AlgorithmJobFactory(creator=user, time_limit=60)
     job_without_perm = AlgorithmJobFactory(time_limit=60)
 
-    job_with_perm.inputs.set([civ1, civ9])
-    job_without_perm.inputs.set([civ2, civ10])
-    job_with_perm.outputs.set([civ3, civ11])
-    job_without_perm.outputs.set([civ4, civ12])
+    job_with_perm.inputs.set([civa1, civb1])

```

Comment:
This giant test I find very hard to follow. It would nice to have explicit tests for making sure that no-one can download the inputs, and only challenge admins can download the outputs.

Reply:
Why would noone be able to download the inputs though? It should be ok for challenge admins to download the inputs, right?

Reply:
True, but we have additional evaluation inputs now which they cannot yet download since they do not get attached to jobs. At the moment, no one is using additional inputs of type file, but if that changes, we might need to enable downloading of inputs for challenge admins. Not relevant now, but I will add a comment about that in the code.